### PR TITLE
Incremented AzCopy version to 10.8.0 in AzureFileCopyV4

### PR DESCRIPTION
**Pull request by amit-avit**
**Task name**: AzureFileCopyV4

**Description**: Incremented AzCopy version to 10.8.0 in AzureFileCopyV4. This was done as there are multiple improvements done from 10.3.3 to this version including the issue raised with us that the previous version is not able to recognise the correct filetype for js files.

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

Validations:
1. Copy to blob succeeded.
2. Copy to VM via blob succeeded.

### CODE REVIEW

**Comment by 20shivangi** [comment]
@amit-avit Please mention in the description , the scenarios that you have tested with this change

File: Tasks/AzureFileCopyV4/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "loc.helpMarkDown": "[Learn more about this task](https://aka.ms/azurefilecopyreadme)",
   "loc.description": "Copy files to Azure Blob Storage or virtual machines",
   "loc.instanceNameFormat": "$(Destination) File Copy",
-  "loc.releaseNotes": "What's new in Version 4.0: <br/>&nbsp;&nbsp; Support AzCopy.exe version 10.3.3",
+  "loc.releaseNotes": "What's new in Version 4.0: <br/>&nbsp;&nbsp; Support AzCopy.exe version 10.8.0",
```
Comment (20shivangi) [inline Tasks/AzureFileCopyV4/Strings/resources.resjson/en-US/resources.resjson:6]: This change must be reflected in all different language files of resources.resjson
Try to build the task , it must automatically be reflected then push the changes to this PR

File: Tasks/AzureFileCopyV4/AzureFileCopyRemoteJob.ps1 [hunk 2]
```diff
@@ -67,7 +67,7 @@ $AzureFileCopyRemoteJob = {
 
                 # Downloading AzCopy from URL and copying it in $azcopyZipPath
                 $webclient = New-Object System.Net.WebClient
-                $webclient.DownloadFile('https://vstsagenttools.blob.core.windows.net/tools/azcopy/10.3/AzCopy.zip',$azCopyZipPath)
+                $webclient.DownloadFile('https://vstsagenttools.blob.core.windows.net/tools/azcopy/10.8/AzCopy.zip',$azCopyZipPath)
 
                 #Unzipping the azcopy zip to $azcopyFolderPath
                 Expand-Archive $azCopyZipPath -DestinationPath $azCopyFolderPath
```
Comment (20shivangi) [inline Tasks/AzureFileCopyV4/AzureFileCopyRemoteJob.ps1:73]: In this file, at line 86, change the version to 10.8.0

File: Tasks/AzureFileCopyV4/AzureFileCopyRemoteJob.ps1 [hunk 3]
```diff
@@ -67,7 +67,7 @@ $AzureFileCopyRemoteJob = {
 
                 # Downloading AzCopy from URL and copying it in $azcopyZipPath
                 $webclient = New-Object System.Net.WebClient
-                $webclient.DownloadFile('https://vstsagenttools.blob.core.windows.net/tools/azcopy/10.3/AzCopy.zip',$azCopyZipPath)
+                $webclient.DownloadFile('https://vstsagenttools.blob.core.windows.net/tools/azcopy/10.8/AzCopy.zip',$azCopyZipPath)
```
Comment (20shivangi) [inline Tasks/AzureFileCopyV4/AzureFileCopyRemoteJob.ps1:70]: In this file, chnage version to 10.8.0 at line 86

Comment (20shivangi) [inline Tasks/AzureFileCopyV4/AzureFileCopyRemoteJob.ps1:70]: In this file, change version to 10.8.0 at line 86

File: Tasks/AzureFileCopyV4/Strings/resources.resjson/en-US/resources.resjson (hunk 1 above)
Comment (20shivangi) [inline Tasks/AzureFileCopyV4/Strings/resources.resjson/en-US/resources.resjson:6]: This change must be reflected in all different language files of resources.resjson
Try to build the task , it must automatically be reflected then push the changes to this PR

Comment (amit-avit) [reply Tasks/AzureFileCopyV4/Strings/resources.resjson/en-US/resources.resjson:6]: I believe it is done by the localisation team and we don't do that manually.

File: Tasks/AzureFileCopyV4/AzureFileCopyRemoteJob.ps1 (hunk 3 above)
Comment (amit-avit) [reply Tasks/AzureFileCopyV4/AzureFileCopyRemoteJob.ps1:70]: done

**Review by 20shivangi — APPROVED** [review]
LGTM

**Review by arjgupta — APPROVED** [review]

**Pull request merged**